packages/compass-import-export/src/modules/import.ts

```diff
@@ -827,9 +875,9 @@ const reducer = (state = INITIAL_STATE, action) => {
 
   if (action.type === FINISHED) {
     const isComplete = state.status !== PROCESS_STATUS.CANCELED;
-    const hasErrors = state.errors.length > 0;
+    const hasErrors = (state.errors || []).length > 0;
 
-    let status = state.stats;
```

Comment:
Updated possible typo, doesn't seem likely to impact anything as a bug as this variable is pretty much always set to something else.